frontend/src/toolbar/experiments/experimentsTabLogic.tsx

```diff
@@ -131,6 +132,12 @@ export const experimentsTabLogic = kea<experimentsTabLogicType>([
                 hideButtonExperiments: () => false,
             },
         ],
+        editSelectorShowing: [
```

Comment:
Nit: If we have a state value that's only used locally within a single element, especially a toggle or boolean value, I strongly prefer using local state instead of a Kea value. In `WebExperimentTransformField.tsx`, we already do this with `[transformSelected, setTransformSelected]`.

While we always aim to separate state from React components, I think this is the one exception. It simplifies working with the value since it only requires a single line of code, and you don't have to update two separate files.

Reply:
Sounds good, updated in 98d0eecf71aacf6d87742a0252ec9f233982da3f